src/Arrays/MPIStateArrays.jl

```diff
@@ -488,7 +488,7 @@ function weighted_norm_impl(Q::SubArray{FT, N, A}, W, ::Val{p}, dims::Colon) whe
       if isfinite(p)
         accum += W[i, 1, k] * abs(Q[i, j, k]) ^ p
       else
-        waQ_ijk = W[i, j, k] * abs(Q[i, j, k])
+        waQ_ijk = W[i, 1, k] * abs(Q[i, j, k])
```

Comment:
Why is this being changed? (And why is the line above read with a `1` also: `accum += W[i, 1, k] * abs(Q[i, j, k]) ^ p`?)

Reply:
weights are set up to have `size(weights, 2) == 1`